client-display/src/common/theme/ThemeHelper.js

```diff
@@ -43,6 +43,43 @@ export default {
       selector: 'body #app .skills-badge .skills-badge-icon, body #app .skills-progress-info-card, body #app .skills-card-theme-border',
       styleName: 'border-color',
     }],
+    pageTitle: {
+      textColor: [{
+        selector: 'body #app .skills-page-title-text-color',
+        styleName: 'color',
+      }, {
+        selector: 'body #app .skills-badge .skills-badge-icon, body #app .skills-progress-info-card, body #app .skills-card-theme-border',
+        styleName: 'border-color',
```

Comment:
should this be here?

Reply:
@mosefer not it shouldn't, i'll remove; good catch!